system/jlib/jlog.cpp

```diff
@@ -2830,39 +2827,38 @@ class DummyLogCtx : implements IContextLogger
     }
     virtual void setGlobalId(const char *id, SocketEndpoint &ep, unsigned pid) override
     {
-        globalId.set(id);
-        appendGloballyUniqueId(localId.clear());
+        logTrace.setGlobalId(id);
     }
     virtual void setCallerId(const char *id) override
     {
-        callerId.set(id);
+        logTrace.setCallerId(id);
     }
-    virtual const char *queryGlobalId() const
+    virtual const char *queryGlobalId() const override
     {
-        return globalId.get();
+        return logTrace.queryGlobalId();
     }
     virtual const char *queryCallerId() const override
     {
-        return callerId.str();
+        return logTrace.queryCallerId();
     }
-    virtual const char *queryLocalId() const
+    virtual const char *queryLocalId() const override
     {
-        return localId.str();
+        return logTrace.queryLocalId();
     }
-    virtual void setHttpIdHeaders(const char *global, const char *caller)
+    virtual void setHttpIdHeaderNames(const char *global, const char *caller) override
     {
         if (global && *global)
```

Comment:
logTrace.setHttpIdHeaderNames(global, caller);

(see comment earlier).  You can then delete the functions setGlobalIdHTTPHeaderName/setCallerIdHTTPHeaderName

The reason is to minimize the public interface